ArraysAmbiguousSNPMiscallingMar2019/allele_flip/flip_alleles.py

```diff
@@ -0,0 +1,71 @@
+import os
+import argparse
```

Comment:
still need `#!/bin/python`, I think

Reply:
and possibly a change in "mod" to make it executable.

Reply:
fixed!